packages/vitest/src/node/config.ts

```diff
@@ -441,6 +441,11 @@ export function resolveConfig(
     resolved.cache.dir = VitestCache.resolveCacheDir(resolved.root, resolved.cache.dir, resolved.name)
 
   resolved.sequence ??= {} as any
+  if (resolved.sequence.shuffle && typeof resolved.sequence.shuffle === 'object') {
+    const { files, tests } = resolved.sequence.shuffle
+    resolved.sequence.sequencer ??= files ? RandomSequencer : BaseSequencer
+    resolved.sequence.shuffle = files || tests
```

Comment:
When `{ files: true, tests: false }`, wouldn't this shuffle test suites/cases?
I guess such pattern is quite unusual, but I thought `resolved.sequence.shuffle = tests` is simply fine here?

Reply:
Good catch!! I've updated it.